backend/Data/Models/Tag.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Data.Models
+{
+    public class Tag
+    {
+        public int Id { get; set; }
+        public string Name { get; set; } = null!;
+        public List<Assigment> Assigments { get; set; } = new List<Assigment>();
```

Comment:
Since the type of `Assignments` is already specified as `List<Assigment>` we can just use the `new()` constructor.

```suggestion
        public List<Assigment> Assigments { get; set; } = new();
```